devtools/cli/src/test/java/io/quarkus/cli/RegistryClientBuilderTestBase.java

```diff
@@ -59,7 +60,8 @@ static void setup() throws Exception {
 
         final BootstrapMavenContext mavenContext = new BootstrapMavenContext(
                 BootstrapMavenContext.config().setWorkspaceDiscovery(false));
-        final Settings settings = getBaseMavenSettings(mavenContext.getUserSettings().toPath());
+        var userSettings = mavenContext.getUserSettings();
+        final Settings settings = Objects.nonNull(userSettings) ? getBaseMavenSettings(userSettings.toPath()) : new Settings();
```

Comment:
```suggestion
        final Settings settings = userSettings != null ? getBaseMavenSettings(userSettings.toPath()) : new Settings();
```

Alternatively, we could pass a `File` to `getBaseMavenSettings(f)` and let it handle it all.

Reply:
Change this to your alternative suggestion.